cst438_registration/src/main/java/com/cst438/domain/EnrollmentRepository.java

```diff
@@ -9,12 +9,16 @@
 public interface EnrollmentRepository extends CrudRepository <Enrollment, Integer> {
 	 
 	@Query("select e from Enrollment e where e.student.email=:email and e.year=:year and e.semester=:semester")
-	public List<Enrollment> findStudentSchedule(
+    List<Enrollment> findStudentSchedule(
 			@Param("email") String email, 
 			@Param("year") int year, 
 			@Param("semester") String semester);
 	
 	@Query("select e from Enrollment e where e.student.email=:email and e.course.course_id=:course_id")
 	Enrollment findByEmailAndCourseId(@Param("email") String email, @Param("course_id") int course_id);
 	
+	@Query("select e from Enrollment e where e.student.student_id=:id")
+    List<Enrollment> findByStudentId( @Param("id") int id); 
+
+	
 }
```

Comment:
Red symbol here is a reminder to add a newline to the end of the file. Formatting your code will add this automatically in most cases.